src/cpl_nuopc/wav_comp_nuopc.F90

```diff
@@ -178,18 +134,14 @@ module wav_comp_nuopc
   use NUOPC_Model           , only : model_label_DataInitialize => label_DataInitialize
   use NUOPC_Model           , only : model_label_SetRunClock    => label_SetRunClock
   use NUOPC_Model           , only : model_label_Finalize       => label_Finalize
-  use NUOPC_Model           , only : NUOPC_ModelGet, SetVM
-  use shr_sys_mod           , only : shr_sys_flush, shr_sys_abort
-  use shr_nl_mod            , only : shr_nl_find_group_name
-  use shr_mpi_mod           , only : shr_mpi_bcast
-  use shr_kind_mod          , only : r8=>shr_kind_r8, i8=>shr_kind_i8, cl=>shr_kind_cl, cs=>shr_kind_cs
-  use shr_file_mod          , only : shr_file_getLogUnit, shr_file_setLogUnit, shr_file_getunit
-  use shr_cal_mod           , only : shr_cal_ymd2date, shr_cal_advDateInt
-  use wav_import_export     , only : advertise_fields, realize_fields, import_fields, export_fields
-  use wav_import_export     , only : state_getfldptr
-  use wav_import_export     , only : wav_coupling_to_cice, wav_coupling_to_mom
-  use wav_shr_methods       , only : chkerr, state_setscalar, state_getscalar, state_diagnose, alarmInit
-  use wav_shr_methods       , only : set_component_logging, get_component_instance, log_clock_advance
+  use NUOPC_Model           , only : NUOPC_ModelGet
+  use wav_wrapper_mod       , only : t_startf, t_stopf, t_barrierf
+  use wav_wrapper_mod       , only : shr_file_getlogunit, shr_file_setlogunit
+  use wav_kind_mod          , only : r8=>shr_kind_r8, i8=>shr_kind_i8, cl=>shr_kind_cl, cs=>shr_kind_cs
+  use wav_import_export     , only : advertise_fields, realize_fields
+  use wav_import_export     , only : state_getfldptr, state_fldchk
+  use wav_shr_mod           , only : chkerr, state_setscalar, state_getscalar, state_diagnose, alarmInit, ymd2date
+  use wav_shr_mod           , only : root_task, stdout, runtype, merge_import, dbug_flag
```

Comment:
Another integer `stdout` is defined below in line 157, causing duplicate definition of `stdout`.

Reply:
Thanks. I've fixed this.